DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/stats/stats.factory.js

```diff
@@ -0,0 +1,40 @@
+﻿(function () {
+    'use strict';
+    var module = angular.module('app.stats');
+
+    module.factory('statsFactory', statsFactory);
+
+    statsFactory.$inject = ['$http'];
+
+    function statsFactory($http) {
+        var baseUrl = 'api/user';
+
```

Comment:
note that vars declared in service can serve you some kind of model. And considering the fact that services are singletones, you can have some kind of single point of truth here. It is the best (and actually the only) option if you will use this data more than in one controller